.github/workflows/skywalking.yaml

```diff
@@ -568,22 +607,26 @@ jobs:
     timeout-minutes: 10
     steps:
       - name: Merge Requirement
+        # check changes, sanity, dependency license, unit, integration, e2e, e2e-istio and e2e-java-versions, 
+        # if all of them are working as expected then naturally exits else return error code
```

Comment:
Exit with different codes in different failures looks a good practice but I don’t think it useful here as the failed tasks will be already red in the workflow so we don’t bother clicking into the detail to see the exit code. And it looks the logic becomes kinda verbose?

Reply:
> Exit with different codes in different failures looks a good practice but I don’t think it useful here as the failed tasks will be already red in the workflow so we don’t bother clicking into the detail to see the exit code. And it looks the logic becomes kinda verbose?

Lemme just change them all into -1, the logic side is explained below.